src/DynamoCore/Graph/Workspaces/LayoutExtensions.cs

```diff
@@ -38,8 +38,11 @@ public static class LayoutExtensions
 
             GenerateCombinedGraph(workspace, isGroupLayout, out layoutSubgraphs, out subgraphClusters);
 
-
-            RecordUndoGraphLayout(workspace, isGroupLayout, reuseUndoRedoGroup);
+            //only record graph layout undo when it is not node autocomplete
```

Comment:
Curious what does this mean?

Reply:
Since we are using the layout method that exists in Dynamo already, it adds an undo for the auto layout. So if we skip it when triggered from node autocomplete it helps minimize the undos